### DISCUSSION TITLE
I cannot create a server

### DISCUSSION THREAD

**Discussion by edvaldoperreira** [qe201020335/BeatTogether-DockerCompose#10] [Q&A]
Hi guys, my server is up, even with the firewall off, I cannot create server on the quest.
I used the default configurations in the json files and docker compose and add the file BeatTogether.cfg inside ModData directory. There is something I am missing out?

**Comment by qe201020335** [comment]
You cannot simply use the json files as is. You have to at least modify the `HostName` in `dedicated.json`. As the comments says, this is the address for the game to connect to. Typically it needs to be the public ip of your server. If you are setting up a LAN server, then this will be the lan ip of the server.

**Comment by edvaldoperreira** [comment]
I did it

I have been configure port forwarding on my router and proxy manager on ubuntu server.

![image](https://github.com/qe201020335/BeatTogether-DockerCompose/assets/11564912/5e958df3-204e-43dd-b197-71274631bdb4)

![image](https://github.com/qe201020335/BeatTogether-DockerCompose/assets/11564912/55668a53-40c8-4713-bddf-35d3b6e1dc2a)

![image](https://github.com/qe201020335/BeatTogether-DockerCompose/assets/11564912/40bf3c49-29b9-4ae5-bcf5-7c8b08bfd24c)

![image](https://github.com/qe201020335/BeatTogether-DockerCompose/assets/11564912/df7a8a27-d552-48b1-ad5f-37e1a5f98deb)

**Reply by qe201020335** [reply]
I see in the logs your dedicated server is using the LAN IP so you can only connect to the server in your LAN. In this case, port forwarding doesn't make sense. 

If you want your server to be accessible outside your LAN, then you need to use your public IP or domain name in `dedicated.json`.
You need to port forward 2328, 23280, 8989, and 30000-40000 (base on your dedicated server settings)
Here is a thread about port forwarding https://github.com/qe201020335/BeatTogether-DockerCompose/discussions/8

**Reply by edvaldoperreira** [reply]
I updated dedicated.json file to my public ip
Add port forward to 2328, 23280, 8989, and 30000-40000

Should I see something in the page domain:2328 ? Because I can see a white page on other ports (23280,8989)
but 2328 seems not working

**Reply by qe201020335** [reply]
There is no page for you to see. None of the ports are for web pages. 
It is also not necessary to use reverse proxy and SSL on the api.
